docs/configurations.md

```diff
@@ -0,0 +1,268 @@
+| `scalar.db.jdbc.admin.connection_pool.max_idle`           | Maximum number of connections that can remain idle in the connection pool for admin.                                                                                         | `10`                         |
+| `scalar.db.jdbc.admin.connection_pool.max_total`          | Maximum total number of idle and borrowed connections that can be active at the same time for the connection pool for admin. Use a negative value for no limit.              | `25`                         |
+
+##### Multi-storage support
+
+ScalarDB supports using multiple storage implementations simultaneously. You can use multiple storages by specifying `multi-storage` as the value for the `scalar.db.storage` property.
+
+For details about using multiple storages, see [Multi-Storage Transactions](multi-storage-transactions.md).
+
+### Use Consensus Commit through ScalarDB Server
+
+[ScalarDB Server](scalardb-server.md) is a standalone server that provides a gRPC interface to ScalarDB. To interact with ScalarDB Server, you must add the following to the ScalarDB properties file:
+
+```properties
+scalar.db.transaction_manager=grpc
+```
+
+The following configurations are available for the gRPC transaction manager for ScalarDB Server:
+
+| Name                                       | Description                                                 | Default                 |
+|--------------------------------------------|-------------------------------------------------------------|-------------------------|
+| `scalar.db.transaction_manager`            | `grpc` should be specified.                                 | -                       |
+| `scalar.db.contact_points`                 | ScalarDB Server host.                                       |                         |
+| `scalar.db.contact_port`                   | Port number for ScalarDB Server.                            | `60051`                 |
+| `scalar.db.grpc.deadline_duration_millis`  | The deadline duration for gRPC connections in milliseconds. | `60000` (60 seconds)    |
+| `scalar.db.grpc.max_inbound_message_size`  | The maximum message size allowed for a single gRPC frame.   | The gRPC default value. |
+| `scalar.db.grpc.max_inbound_metadata_size` | The maximum size of metadata allowed to be received.        | The gRPC default value. |
+
+For details about ScalarDB Server, see [ScalarDB Server](scalardb-server.md).
+
+## ScalarDB Server configurations
+
+[ScalarDB Server](scalardb-server.md) is a standalone server that provides a gRPC interface to ScalarDB. This section explains ScalarDB Server configurations.
+
+In addition to the [configurations when using Consensus Commit through ScalarDB Server](#use-consensus-commit-through-scalardb-server) and [other ScalarDB configurations](#other-scalardb-configurations), the following configurations are available for ScalarDB Server:
```

Comment:
```suggestion
In addition to the [configurations when using Consensus Commit directly](#use-consensus-commit-directly) and [other ScalarDB configurations](#other-scalardb-configurations), the following configurations are available for ScalarDB Server:
```
Isn't this a mistake since ScalarDB Server is the direct client for databases?

@josh-wong @brfrn169

Reply:
Yes, that was a mistake. ScalarDB Server should directly use Consensus Commit. Thank you for pointing it out!

Reply:
Thank you for catching this! I've applied your suggestion.